packages/ui-buttons/src/BaseButton/types.ts

```diff
@@ -25,8 +25,9 @@
 import { AsElementType } from '@instructure/shared-types'
 import type { InteractionType } from '@instructure/ui-react-utils'
 import type { Spacing } from '@instructure/emotion'
+import React from 'react'
 
-export type BaseButtonProps = {
+export type BaseButtonProps = React.PropsWithChildren<{
```

Comment:
Just a note: I found it better to list children among the props instead of using this type, because
- it is easier to read
- and sometimes we have to manually type it anyway (e.g. it is required or not just simply "ReactNode")